api/middleware/logger.go

```diff
@@ -129,14 +138,9 @@ func (entry *apiLoggerEntry) writePanic(status, bytes int, elapsed time.Duration
 	log.Int("http.content_length", bytes)
 	log.Int("elapsed_time_ms", int(elapsed.Milliseconds()))
 
-	fmt.Fprintf(entry.buf, "%03d", status)
-	fmt.Fprintf(entry.buf, " %dB", bytes)
-	entry.buf.WriteString(" in ")
-	fmt.Fprintf(entry.buf, "%s", elapsed)
-	fmt.Fprintf(entry.buf, " - Panic: %+v", v)
-	entry.buf.WriteString("\n")
-	entry.buf.WriteString(string(stack))
-	log.Error(entry.buf.String())
+	log.String("stackTrace", string(stack))
+
+	log.Error(fmt.Sprintf("%s: panic", entry.msg))
```

Comment:
hm, seems this construct now should be present with our new logger?

Reply:
Fixed